pages/controls/Certificate_and_Public_Key_Pinning.md

```diff
@@ -315,25 +315,11 @@ Java, iOS, .Net, and OpenSSL.
 
 ### HTTP pinning
 
-[RFC 7469](http://www.rfc-editor.org/rfc/rfc7469.txt) introduced a new
-HTTP header that allows SSL servers to declare hashes of their
-certificates with time scope in which these certificates should not be
-changed. For example:
-
-    Public-Key-Pins: max-age=2592000;
-    pin-sha256="E9CZ9INDbd+2eRQozYqqbQ2yXLVKB9+xcprMF+44U1g=";
-    pin-sha256="LPJNul+wow4m6DsqxbninhsWHlwfp0JecwQzYpOLmCQ=";
-    report-uri="http://example.com/pkp-report"
-
-Please note that [RFC 7469](http://www.rfc-editor.org/rfc/rfc7469.txt)
-is controversial since it allows overrides for locally installed
-authorities. That is, it allows an adversary or other party who
-successfully phishes the user to override a known good pinset with
-non-authentic or fraudulent information. Second, the reporting mechanism
-is suppressed from broken pinsets, so a complying user agent will be
-complicit in the cover up after the fact. That is, the reporting of the
-broken pinset is called out as **MUST NOT** report
-[\[1](https://en.wikipedia.org/w/index.php?title=HTTP_Public_Key_Pinning)\].
+[Expect-CT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT) header allows sites to opt in to the [Certificate Transparency](https://developer.mozilla.org/en-US/docs/Web/Security/Certificate_Transparency) framework, in report or enforcement mode, based on the readiness of the application.
```

Comment:
Good additions! To make it more complete: Maybe it is good to add a little bridge here: e.g. that http pinning (with its header) was where it started, but with the flaws found in the implementation ending up in the deprecation and removal of HPKP support in major browsers and that now expect-ct is the new kid?

Reply:
Noted, I will send in an update accordingly.

Reply:
Added a note on HPKP with it being replaced by CT and the header. Let me know what you think about it.